src/Routes.js

```diff
@@ -36,6 +36,11 @@ const Routes = ({ authUser }) => {
         <Route exact path={ ROUTES.PROJECTS } component={ Projects } />
         <Route exact path={ ROUTES.WORKSPACE } component={ Workspace } />
         <Route exact path={ ROUTES.PAPER_EDITOR } component={ PaperEditor } />
+        <Route
```

Comment:
I wonder if it's worth adding and leaving a comment about why we need two paths? Saw your note about 'gotchas' but might confuse folks working on the project.

Reply:
> This looks good to me, Eimi - just a small question about timestamps on project creation. Also just curious about the "converted from" field on the transcripts now - does that ever populate? Lol.

it does now 😄